Packs/PAN-OS/Playbooks/PAN-OS_-_Add_Anti-Spyware_Security_Profile_To_Rule.yml

```diff
@@ -0,0 +1,1029 @@
+id: PAN-OS - Add Anti-Spyware Security Profile To Rule
+version: -1
+name: PAN-OS - Add Anti-Spyware Security Profile To Rule
+description: |-
+  This playbook is used to add an Anti-Spyware security profile to a security rule in PAN-OS in a safe manner: it provides granular control over the behavior for cases where a rule already has an Anti-Spyware profile attached, or has a security profile group configured to it with/without an Anti-Spyware profile.
+  The playbook outputs the Anti-Spyware profile configured for the rule at the time the playbook finished. This could be the previous profile if it was not overwritten, or it could be a new one that it was overwritten with or has just been added.
+tags:
+- PAN-OS
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 03fa6d8d-2cb3-459e-800d-4ab89475bc4d
+    type: start
+    task:
+      id: 03fa6d8d-2cb3-459e-800d-4ab89475bc4d
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "2"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 70
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "1":
+    id: "1"
+    taskid: a65e9138-07cf-4003-82f8-e223e6d6ac59
+    type: condition
+    task:
+      id: a65e9138-07cf-4003-82f8-e223e6d6ac59
+      version: -1
+      name: Does it have an anti-spyware profile already?
+      description: Checks whether the rule already has an anti-spyware profile attached.
```

Comment:
The conditional task is more complicated then the description provided. Maybe try to elaborate on the conditions?

Reply:
I agree regarding the confusing labels and the title missing context, fixed those